modules/ref/pages/client-settings.adoc

```diff
@@ -46,6 +46,11 @@ Default: `best_effort_retry_strategy{ controlled_backoff }`
 +
 The retry strategy decides if an operation should be retried or canceled. While implementing a custom strategy is fairly advanced, the SDK ships with a best effort retry strategy out of the box (`best_effort_retry_strategy`). This default will retry the operation (when the error is retriable) until it either succeeds or the maximum request lifetime is reached.
 
+Name: *Zone Aware Replica Reads*::
```

Comment:
If we want to call attention to this concept, we can put it in the description, but again, we have been trying to be careful to call it "Preferred Server Group Replica Read" or "Replica Read from Preferred Server Group"